index.js

```diff
@@ -34,58 +35,65 @@ module.exports = {
   description,
 
   install(options) {
-    if (fs.existsSync(path.join('..', 'package.json'))) {
-      options.ui.writeInfoLine(
-        "Existing monorepo detected! The blueprint will only create the addon and test-app folders, and omit any other files in the repo's root folder."
-      );
+    if (!options.addonOnly) {
+      if (fs.existsSync(path.join('..', 'package.json'))) {
```

Comment:
as an aside, I think this check should probably use `findUp` -- will PR that later